PWGHF/Core/HfHelper.h

```diff
@@ -82,6 +82,63 @@ class HfHelper
     return candidate.cosThetaStar(std::array{o2::analysis::pdg::MassKPlus, o2::analysis::pdg::MassPiPlus}, o2::analysis::pdg::MassD0, 0);
   }
 
+  // D*(±) --> D0(bar) + π(±)
+  struct D0FromDstar {
+    template <typename T>
+    auto ctD0(const T& candidate)
+    {
+      return candidate.d0ct(o2::analysis::pdg::MassD0);
+    }
+
```

Comment:
Are these needed? Also, this class is temporary. It would be better if you could implement these directly as dynamic columns.

Reply:
Earlier I did not used this class, but seeing that it is used in 2-prong selector, I used this. now I will revert back.

Reply:
reverted back